app/page.tsx

```diff
@@ -72,6 +72,7 @@ export default function Home() {
             onClick={() => setIsQuizDialogOpen(true)}
             variant="secondary"
             className="gap-2"
+            // disabled={cards.length === 0}
```

Comment:
Bug #1:

The button should ideally be disabled when there are no flashcards to quiz on `(cards.length === 0)`